.gitlab-ci.yml

```diff
@@ -86,10 +84,24 @@ default:
     - rustup +nightly show
     - cargo +nightly --version
 
+.rusty-cachier:
+  before_script:
+    - curl -s https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.parity.io/parity/infrastructure/ci_cd/rusty-cachier/client/-/raw/release/util/install.sh | bash
+    - rusty-cachier environment check --gracefully
+    - $(rusty-cachier environment inject)
+    - rusty-cachier project mtime
+  after_script:
+    - rusty-cachier snapshot destroy
```

Comment:
this might be not needed, or is it against being randomly reused if some runner picks the same container up?

Reply:
It deletes overlay leftovers (e.g. `upperdir` & `workdir`).